permits/models.py

```diff
@@ -1040,9 +1040,21 @@ class PermitRequestValidation(models.Model):
     validation_status = models.IntegerField(
         _("Statut de validation"), choices=STATUS_CHOICES, default=STATUS_REQUESTED
     )
-    comment_before = models.TextField(_("Commentaires (avant)"), blank=True)
-    comment_during = models.TextField(_("Commentaires (pendant)"), blank=True)
-    comment_after = models.TextField(_("Commentaires (après)"), blank=True)
+    comment_before = models.TextField(
+        _("Commentaire (avant)"),
+        blank=True,
+        help_text=_("Facultatif, sera transmis uniquement au service pilote"),
+    )
+    comment_during = models.TextField(
+        _("Commentaire (pendant)"),
+        blank=True,
+        help_text=_("Facultatif, sera transmis uniquement au service pilote"),
+    )
+    comment_after = models.TextField(
+        _("Commentaire (après)"),
+        blank=True,
+        help_text=_("Facultatif, sera transmis uniquement au service pilote"),
+    )
```

Comment:
Validations comments are visible for user, thus I would change to "Facultatif" only

Reply:
user can anyway see it in default print so, I would remove "sera transmis uniquement au service pilote"

Reply:
On my side I cannot see the comments as user in the summary, let's investigate it before squashing migrations.

Reply:
I confirm it's not visible to the user. But it might be in the future. The safer would be for now not to mention this